README.md

```diff
@@ -2,8 +2,7 @@
 ## Utilize Kubernetes Service-Catalog to dynamically provision CNS Object Storage.
 
 ## WARNING!
-The work in this project is a proof of concept and not intended for anything
-more than that.
+The work in this project is a proof of concept and not intended production.
 
 ## Overview
 A core feature of the Kubernetes system is the ability to provision a diverse
```

Comment:
"... on demand via [dynamic provisioning](http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html)."